components/FavPokemon.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, View, Image, Button } from 'react-native';
+
+const FavPokemon = ({ navigation }) => {
+  const [favPokemon, setFavPokemon] = useState(null);
+  const [loading, setLoading] = useState(true);
+
+  const getFavPokemon = async favPokemonName => {
+    try {
+      const res = await fetch(
+        `https://pokeapi.co/api/v2/pokemon/${favPokemonName}`
+      );
+      const data = await res.json();
+
+      setFavPokemon(data);
+      setLoading(false);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    getFavPokemon('pikachu');
+    //eslint-disable-next-line
+  }, []);
+
+  if (loading) {
+    return (
+      <View style={styles.container}>
+        <Text>Loading...</Text>
+      </View>
+    );
+  } else {
+    return (
+      <View style={styles.container}>
+        <Text style={styles.header}>This is {favPokemon.name}!</Text>
+        <Image
+          style={styles.image}
+          source={{ uri: favPokemon.sprites.front_default }}
+        />
+        <View>
+          <View style={styles.propRow}>
+            <Text style={styles.properties}>Type: </Text>
+            <Text style={styles.properties}>
+              {favPokemon.types.map(type => type['type']['name'])}
+            </Text>
+          </View>
+          <View style={styles.propRow}>
+            <Text style={styles.properties}>Weight: </Text>
+            <Text style={styles.properties}>{favPokemon.weight}</Text>
+          </View>
+          <View style={styles.propRow}>
```

Comment:
You could think if it would be worth to extract it to a separate component:
```ts
const PropRow = ({ leftLabel, rightLabel }) => {
...
}
```
as you use the same structure (propRow + 2xproperties text) 4 times here